.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: required
 
 language: go
-go_import_path: github.com/uber/jaeger-client-go
+go_import_path: github.com/uber/jaeger-client-go/v2
```

Comment:
well, thats an easy search&replace for me, so i can do that :)

Reply:
done, that will have some problems, until a v3.x.x tag exists :)

Reply:
The changes needed in jaeger-lib are a little bit more complicated. I'd say that can be done, after adding support here, as it will take a little longer to fix every issue there

Reply:
Can you elaborate? Why is lib more difficult?

Reply:
Renamed imports make it difficult :D